herepy/__init__.py

```diff
@@ -6,7 +6,7 @@
 __email__ = "abdullahselek.os@gmail.com"
 __copyright__ = "Copyright (c) 2017 Abdullah Selek"
 __license__ = "MIT License"
-__version__ = "3.6.4"
+__version__ = "3.6.5"
```

Comment:
I don't think we need to bump the version as `3.6.4` is not released yet.

Reply:
Sorry! I also bumped the version in #82 ...